easybuild/easyconfigs/g/gSOAP/gSOAP-2.8.47-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "ConfigureMake"
+
+name = 'gSOAP'
+version = '2.8.47'
+
+homepage = 'http://www.cs.fsu.edu/~engelen/soap.html'
+description = """The gSOAP toolkit is a C and C++ software development toolkit for
+ SOAP and REST XML Web services and generic C/C++ XML data bindings.
+ The toolkit analyzes WSDLs and XML schemas (separately or as a combined set) and maps the XML schema types
+ and the SOAP/REST XML messaging protocols to easy-to-use and efficient C and C++ code.
+ It also supports exposing (legacy) C and C++ applications as XML Web services
+ by auto-generating XML serialization code and WSDL specifications.
+ Or you can simply use it to automatically convert XML to/from C and C++ data.
+ The toolkit supports options to generate pure ANSI C or C++ with or without STL."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.3'),
+]
+
+dependencies = [('zlib', '1.2.11')]
+
+sources = ['%(namelower)s_%(version)s.zip']
+source_urls = [SOURCEFORGE_SOURCE]
```

Comment:
Use the direct link:

```python
source_urls = ['https://downloads.sourceforge.net/project/gsoap%(version_major)s/gsoap-%(version_major_minor)s/']
```

There's on update since yesterday btw, 2.8.48 ;)